### PR TITLE
Add raw version of notebook for name suggestion

### PR DESCRIPTION
**Pull request by EgorBu**
Signed-off-by: egor <egor@sourced.tech>

### CODE REVIEW

File: notebooks/Name suggestion.ipynb [hunk 1]
```diff
@@ -0,0 +1,1943 @@
+{
```
Comment (m09) [inline notebooks/Name suggestion.ipynb:1]: <p>It seems we are duplicating a bit <code>utils.py</code>'s <code>gitbase_query</code>, probably we can merge both if we add the necessary logic to one of them!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by bzz** [comment]
As Egor will not have time today to spend on this much - I suggest merging it ASAP as is, and addressing the review later. @m09 WDYT?

**Comment by m09** [comment]
Fine to merge yes :+1:

**Review by m09 — APPROVED** [review]
We're almost there now :checkered_flag: thanks @EgorBu

File: notebooks/Name suggestion.ipynb (hunk 1 above)
Comment (EgorBu) [reply notebooks/Name suggestion.ipynb:1]: yep, I copy-pasted it from tensorflow analysis. But we can definitely use the same

**Pull request merged**